tests/test_models/test_nn/deepar_native/test_deepar_native.py

```diff
@@ -0,0 +1,108 @@
+from unittest.mock import MagicMock
+
+import numpy as np
+import pytest
+from pytorch_lightning import seed_everything
+
+from etna.metrics import MAE
+from etna.models.nn import DeepARNativeModel
+from etna.models.nn.deepar_native.deepar import DeepARNativeNet
+from etna.models.nn.deepar_native.loss import GaussianLoss
+from etna.models.nn.deepar_native.loss import NegativeBinomialLoss
+from etna.pipeline import Pipeline
+from etna.transforms import StandardScalerTransform
+from tests.test_models.utils import assert_model_equals_loaded_original
+from tests.test_models.utils import assert_sampling_is_valid
+
+
+@pytest.mark.parametrize(
+    "horizon,loss,transform,epochs,lr,eps",
+    [
+        (8, GaussianLoss(), [StandardScalerTransform(in_column="target")], 100, 1e-3, 0.05),
+        (13, GaussianLoss(), [StandardScalerTransform(in_column="target")], 100, 1e-3, 0.05),
+        (15, GaussianLoss(), [StandardScalerTransform(in_column="target")], 100, 1e-3, 0.05),
+        (8, NegativeBinomialLoss(), [], 300, 1e-2, 0.05),
+        (13, NegativeBinomialLoss(), [], 300, 1e-2, 0.06),
+        (15, NegativeBinomialLoss(), [], 300, 1e-2, 0.05),
+    ],
+)
+def test_deepar_model_run_weekly_overfit(
+    ts_dataset_weekly_function_with_horizon, horizon, loss, transform, epochs, lr, eps
+):
+    """
+    Given: I have dataframe with 2 segments with weekly seasonality with known future
+    Then: I get {horizon} periods per dataset as a forecast and they "the same" as past
+    """
+    seed_everything(0, workers=True)
+    ts_train, ts_test = ts_dataset_weekly_function_with_horizon(horizon)
+    encoder_length = 14
+    decoder_length = 14
+    model = DeepARNativeModel(
+        input_size=1,
+        encoder_length=encoder_length,
+        decoder_length=decoder_length,
+        scale=False,
```

Comment:
By the way, does `scale=True` works ok(in terms of metric) for NegativeBinomialLoss?

Reply:
works not ok